tango_simlib/tango_sim_generator.py

```diff
@@ -350,7 +365,14 @@ def initialize_dynamic_attributes(self):
                             MODULE_LOGGER.info(
                                 "No setter function for " + prop + " property")
                     attr.set_default_properties(attr_props)
-                    self.add_attribute(attr, self.read_attributes)
+
+                    if rw_type == AttrWriteType.READ:
+                        self.add_attribute(attr, self.read_attributes)
+                    elif rw_type in (AttrWriteType.READ_WRITE,
+                                     AttrWriteType.READ_WITH_WRITE, AttrWriteType.WRITE):
+                        self.add_attribute(
+                            attr, self.read_attributes, self.write_attributes)
```

Comment:
If it is `AttrWriteType.WRITE`, then should it get `read_attributes`?

Reply:
No it shouldn't. I have removed it.